.github/workflows/linters.yml

```diff
@@ -28,10 +28,10 @@ jobs:
           node-version: "12.x"
       - name: Setup Webhint
         run: |
-          npm install --save-dev hint@6.x
-          [ -f .hintrc ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css/.hintrc
+          npm install --save-dev hint@6.1.4
+          [ -f .hintrc ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css-js/.hintrc
       - name: Webhint Report
-        run: npx hint --telemetry=off 
+        run: npx hint .
```

Comment:
**Kindly fix the errors related to your linters**
_After install locally your project i ran the next installation:_
`` npm install --save-dev hint@6.x ``
and then:
`` npx hint . ``
This is the result displayed:
![Screen Shot 2021-11-08 at 20 06 28](https://user-images.githubusercontent.com/22258274/140849253-3cdc0636-c6c8-4b45-ac0c-88395177daa6.png)
 
- Kindly fix your issues and double check your linters before to request another code review.

I hope that this information would be helpful, if you have any question you can write me on slack.